encoding/x/protobuf/inbound.go

```diff
@@ -81,13 +81,11 @@ func (u *unaryHandler) Handle(ctx context.Context, transportRequest *transport.R
 	var wireError *wirepb.Error
 	if appErr != nil {
 		responseWriter.SetApplicationError()
-		wireError = &wirepb.Error{
-			appErr.Error(),
-		}
+		wireError = &wirepb.Error{Message: appErr.Error()}
```

Comment:
I dont want to do composite keys on purpose to verify at compile time if the message is completely filled out appropriately @sectioneight